cmd/registry-server/main.go

```diff
@@ -38,6 +38,8 @@ func main() {
 		if err != nil {
 			log.Fatalf("%s", err.Error())
 		}
+	} else {
+		log.Fatalf("Config file name must be provided using the -c flag")
```

Comment:
I don't understand the issue with containers well enough to document it. Do you have any suggestions?

Reply:
We can more fully address this after this change is in, but for background the Dockerfiles build registry-server with CGO disabled (to produce static binaries with no dynamic dependencies) and this means that the SQLite backend is unavailable in containers. It wouldn't make sense to use it there anyway because containers are usually ephemeral (unless some external storage is mounted into them). The Dockerfiles also run the registry-server with "-c config/registry.yaml" - we can change this to "-c config/postgres.yaml" (possibly deleting registry.yaml altogether) and we'll need to document in the main README that this postgres.yaml should be configured to point to a running instance of Postgres (with an appropriate database name, username, and password in postgres.yaml)